java/11/Dockerfile

```diff
@@ -10,5 +10,31 @@ FROM base as base-arm64
 ARG AWSCLI_ARCH="linux-aarch64"
 
 FROM base-${TARGETARCH}
-COPY run.sh .
-RUN sh run.sh
+
+# Updating packages using sources
+RUN cat /etc/apt/sources.list && \
+    apt-get -qq clean -qq && apt-get -qq update && \
+
+# Install base
+    apt-get -qq -y install build-essential curl git make openssh-client jq unzip groff less && \
+
+# Installing AWS CLIv2
+    curl https://awscli.amazonaws.com/awscli-exe-${AWSCLI_ARCH}.zip -o awscliv2.zip && \
+    unzip -q awscliv2.zip && \
+    ./aws/install && \
+    rm -f awscliv2.zip && \
+    rm -rf aws && \
+
+# Install graphviz
+    apt-get -qq -y install graphviz && \
+
+# Install postgresql-client
+    apt-get -qq -y install postgresql-client && \
```

Comment:
Maybe installing `graphviz` and `postgresql-client` with the same command would be slightly faster, since it won't read the dpkg database twice. It could even be grouped with the first install command.

That can be done in another PR, though.